File: fixture/Aspect/input/Execution/Fibonacci_expectation_after.js
function fibonacci(n) {
	if (n < 2) {
		{
			Console.log("Inserted after.");
		}
		return n;
	}
	else {
		{
			Console.log("Inserted after.");
		}
		return fibonacci(n - 1) + fibonacci(n - 2);
	}
}

File: fixture/Aspect/input/Execution/Fibonacci_expectation_after.java
public class Fibonacci {
	public int fibonacci(int n) {
		if(n < 2) {
			{
				System.out.println("Inserted after.");
			}
			return n;
		}
		else {
			{
				System.out.println("Inserted after.");
			}
			return fibonacci(n - 1) + fibonacci(n - 2);
		}
	}
}